assignment4/IslandCounterTest.java

```diff
@@ -0,0 +1,35 @@
+package com.katermar.assignment4;
+
+import java.util.Arrays;
+
+/**
+ * Created by katermar on 6/21/2017.
+ */
+public class IslandCounterTest {
+
+    public static void main(String args[]) {
+        char map[][] = {{'F', 'T', 'F', 'T'},
+                        {'T', 'T', 'F', 'F'},
+                        {'F', 'F', 'T', 'F'},
+                        {'F', 'F', 'T', 'F'}};
+        char emptyArray[][] = {{}, {}};
+        testIslandCounter(3, map);
+        testIslandCounter(2, map);

```

Comment:
The second test always fails. The PR should contain only passing tests, so that they can be used to easily check if code works.